tex/generic/pgf/frontendlayer/tikz/libraries/tikzlibrarycalendar.code.tex

```diff
@@ -358,7 +358,7 @@
 \def\tikz@lib@cal@calendar{%
   \begingroup%
     \let\tikz@lib@cal@ifs=\pgfutil@empty%
-    \tikz@expandcount=1000\relax%
```

Comment:
1000 seemed really excessive.  It was probably set to 1000 because it would never be reset.